### PR TITLE
PUD-910: Update from swagger. Cleanup null status checks.

### PR DESCRIPTION
**Pull request by philsegal**

### CODE REVIEW

File: server/@types/manage-recalls-api/models/RecallResponse.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export type RecallResponse = {
     sentenceExpiryDate?: string;
     sentenceLength?: SentenceLengthRes;
     sentencingCourt?: string;
-    status?: RecallResponse.status;
+    status: RecallResponse.status;
```
Comment (jonnywyatt) [inline server/@types/manage-recalls-api/models/RecallResponse.ts:58]: maybe worth updating the get-recall.json response in fake-manage-recalls-api

Comment (philsegal) [reply server/@types/manage-recalls-api/models/RecallResponse.ts:58]: yep, that was causing the failure :(

File: server/routes/handlers/helpers/documents.ts [hunk 2]
```diff
@@ -215,8 +215,5 @@ export const enableDeleteDocuments = (recallStatus: RecallResponse.status, urlIn
   if (recallStatus !== RecallResponse.status.BEING_BOOKED_ON) {
     return false
   }
-  if (urlInfo.fromPage && urlInfo.fromPage !== 'check-answers') {
-    return false
-  }
-  return true
+  return !(urlInfo.fromPage && urlInfo.fromPage !== 'check-answers')
```
Comment (jonnywyatt) [inline server/routes/handlers/helpers/documents.ts:218]: This is shorter but less readable to me 🤷‍♂️

**Review by jonnywyatt — APPROVED** [review]

File: server/routes/handlers/helpers/documents.ts (hunk 2 above)
Comment (philsegal) [reply server/routes/handlers/helpers/documents.ts:218]: oh sorry, didn't mean to include it. Shall I revert?

**Pull request merged**